test/browser/column/resizable.test.tsx

```diff
@@ -229,14 +233,31 @@ test('should remeasure flex columns when resizing a column', async () => {
     onColumnResize
   });
   const grid = getGrid();
-  await expect.element(grid).toHaveStyle({ gridTemplateColumns: '639.328px 639.328px 639.344px' });
+
+  function testGridTemplateColumns(chrome: string, firefox: string, firefoxCI = firefox) {
```

Comment:
Is there a better way? I initially thought about using [`toBeCloseTo`](https://vitest.dev/api/expect.html#tobecloseto) but FF in CI has completely different column size. No idea why